/**:
  ros__parameters:
    timer_hz: 50.0
    csv_path: /path/to/lane.csv
    lidar_frame_id: laser
    fov_deg: 270.0
    max_range: 30.0
    num_rays: 1080
    range_min: 0.001

    debug: true
    map_frame_id: map
